app/views/contribute/canonical_data.erb

```diff
@@ -67,7 +67,7 @@
             <% implementations.each do |implementation| %>
               <ul>
                 <li>
-                <a href="<%= [implementation.repo, implementation.exercise_dir].join("/") %>">
+                <a href="<%= [implementation.track.repository, implementation.send(:exercise_dir)].join("/") %>">
```

Comment:
There is a public method for this:
`                <a href="<%= implementation.git_url %>">`